rpmautospec/subcommands/convert.py

```diff
@@ -176,14 +176,19 @@ def convert_to_autochangelog(self):
         log.debug("Split %d lines to 'changelog' file", len(self.changelog_lines))
         self.converted_changelog = True
 
-    def commit(self, message: Optional[str] = None):
+    def commit(self, message: Optional[str] = None, signoff: bool = False):
         if self.repo is None:
             log.debug("Unable to open repository at '%s'", self.path)
             return
 
         if message is None:
             message = self.describe_changes(for_git=True) + "\n\n[skip changelog]"
 
+        signature = self.repo.default_signature
```

Comment:
Is this guaranteed to exist? Should we test for it and throw an error if it's unavailable?